boot/bootutil/src/loader.c

```diff
@@ -185,11 +201,17 @@ boot_status_source(void)
              table->bst_magic_scratch   == state_scratch.magic) &&
             (table->bst_copy_done_slot0 == 0    ||
              table->bst_copy_done_slot0 == state_slot0.copy_done)) {
-
-            return table->bst_status_source;
+            uint8_t source = table->bst_status_source;
```

Comment:
Ah, lovely, a coding standard doc! I didn't find one in tree and wasn't quite sure. I'll fix this up to match the mynewt style and make the other changes.

On a related note, how would you feel about adding a hidden file to the top level that directs emacs to format things according to mynewt style? It'd be called .dir-locals.el.

If that's undesirable, how about a patch adding the same to .gitignore, so I can have one in my clone?